service/api/src/test/java/org/eclipse/kapua/KapuaRuntimeExceptionTest.java

```diff
@@ -0,0 +1,272 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(JUnitTests.class)
+public class KapuaRuntimeExceptionTest extends Assert {
+
+    String expectedErrorMessage;
+    KapuaErrorCode kapuaErrorCode;
+    Object argument1, argument2, argument3;
+    Throwable[] throwables;
+
+    @Before
+    public void initialize() {
+        expectedErrorMessage = "kapua-service-error-messages";
+        kapuaErrorCode = KapuaErrorCodes.ADMIN_ROLE_DELETED_ERROR;
+        argument1 = "user";
+        argument2 = 1;
+        argument3 = 'c';
+        throwables = new Throwable[]{new Throwable(), null};
+    }
+
```

Comment:
Please delete empty line.

Reply:
I used index i to get values from different arrays in the same position